charts/argo/templates/workflow-controller-sa.yaml

```diff
@@ -2,3 +2,5 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ .Values.controller.serviceAccount }}
+  annotations:
+{{ toYaml .Values.controller.serviceAccountAnnotation | indent 4 }}
```

Comment:
What about using `controller.serviceAccount.annotations`?

Reply:
Agree that that would make more sense. Currently in the values.yaml `controller.serviceAccount` is reserved for the name `string`, so changing that wouldn't be backwards compatible. Is this what you had in mind:
```
controller:
    serviceAccount:
      name: argo 
      annotations: ~ 
```

Reply:
Makes sense we don't want to do anything breaking, perhaps we just use `serviceAccountAnnotations` and call it a day.